easy_profiler_core/CMakeLists.txt

```diff
@@ -233,19 +233,12 @@ if (MSVC)
     target_compile_options(easy_profiler PRIVATE /WX)
 endif ()
 
-if (APPLE)
-    target_compile_options(easy_profiler PUBLIC -std=gnu++11)
```

Comment:
I removed this completely because it was already covered by either the old L241 or the old L247